.github/workflows/regression-report.yml

```diff
@@ -81,30 +99,177 @@ jobs:
+            CONFIG_BUNDLE=$(echo "$COMMENT_BODY" | \
             sed -E 's/.*(Github|GitHub), generate report for //g')
-          echo "CONFIG_BUNDLE_PATH=$CONFIG_BUNDLE_PATH"
-          ./.ci/append-to-github-output.sh "config_bundle_path" "$CONFIG_BUNDLE_PATH"
+            ./.ci/append-to-github-output.sh "config_bundle" "$CONFIG_BUNDLE"
+          else
+            echo "No recognized pattern found in COMMENT_BODY."
+            exit 1
+          fi
+          
+          echo "Parsing completed successfully"
+
+      - name: Set branch
+        id: set_branch
+        env:
+          PULL_REQUEST_URL: ${{ github.event.issue.pull_request.url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl --fail-with-body -X GET "${PULL_REQUEST_URL}" \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: token $GITHUB_TOKEN" \
+              -o .ci-temp/info.json
+          jq --raw-output .head.ref .ci-temp/info.json > .ci-temp/branch
+          jq --raw-output .head.sha .ci-temp/info.json > .ci-temp/commit_sha
+          echo "ref=$(cat .ci-temp/branch)" >> $GITHUB_OUTPUT
+          echo "commit_sha=$(cat .ci-temp/commit_sha | cut -c 1-7)" >> $GITHUB_OUTPUT
+
+
+  handle_config_bundle:
+    needs: parse_comment
+    if: needs.parse_comment.outputs.mode == 'use_config_bundle'
+    runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.set_links.outputs.config_link }}
+      projects_link: ${{ steps.set_links.outputs.projects_link }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
 
       - name: Set config and projects links
         id: set_links
         run: |
           BASE_URL="https://raw.githubusercontent.com/checkstyle/test-configs/main"
-          CONFIG_BUNDLE_PATH="${{ steps.parse.outputs.config_bundle_path }}"
-          CONFIG_LINK="$BASE_URL/$CONFIG_BUNDLE_PATH/config.xml"
-          PROJECTS_LINK="$BASE_URL/$CONFIG_BUNDLE_PATH/list-of-projects.properties"
-          echo "CONFIG_LINK=$CONFIG_LINK"
-          echo "PROJECTS_LINK=$PROJECTS_LINK"
+          CONFIG_BUNDLE="${{ needs.parse_comment.outputs.config_bundle }}"
+          CONFIG_LINK="$BASE_URL/$CONFIG_BUNDLE/config.xml"
+          PROJECTS_LINK="$BASE_URL/$CONFIG_BUNDLE/list-of-projects.properties"
           ./.ci/append-to-github-output.sh "config_link" "$CONFIG_LINK"
           ./.ci/append-to-github-output.sh "projects_link" "$PROJECTS_LINK"
 
-      - name: Set branch
+  handle_generate_config:
+    needs: parse_comment
+    if: needs.parse_comment.outputs.mode == 'generate_config'
+    runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.generate_config.outputs.config_link }}
+      projects_link: ${{ steps.generate_config.outputs.projects_link }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Checkout Checkstyle repository
```

Comment:
Why you need master branch ?

Reply:
I am checking out checkstyle's master branch to search for the input files.  Without this, find input files fails.

Reply:
you need to find Inputs in PR branch not a master, as PR can introduce new Input.
Please confirm it works like this.

Reply:
moved to new item